rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMService.java

```diff
@@ -74,12 +74,13 @@
      * Detailed info about the nodes currently held by the RM
      * presented as two arrays of nodes and nodesources referencing each others
      * @param sessionId current session
+     * @param clientCounter latest counter client is aware of
      * @return a JSON object containing two arrays named nodesList and nodeSources, that contain all info about current
      * 		nodes and nodesources in the RM
      * @throws RestServerException 
      * @throws ServiceException
      */
-    String getMonitoring(String sessionId) throws RestServerException, ServiceException;
+    String getMonitoring(String sessionId, Long clientCounter) throws RestServerException, ServiceException;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)